### PR TITLE
Improved build pipeline display when pipeline stages wrap

### PR DESCRIPTION
**Pull request by rhamilto**
320px
400px
600px
800px
1000px
1200px
1600px

### CODE REVIEW

File: app/views/directives/build-pipeline.html [hunk 1]
```diff
@@ -24,27 +24,27 @@
...
-            <div class="pipeline-time" ng-class="stage.status" ng-if="!stage.durationMillis && !(stage | pipelineStagePendingInput)">not started</div>
+    </div>
+    <div class="pipeline">
+      <div class="pipeline-stage" ng-repeat="stage in jenkinsStatus.stages track by stage.id">
+        <div column class="pipeline-stage-column">
+          <div class="pipeline-stage-name" ng-class="build.status.phase">
+            {{stage.name}}
           </div>
+          <pipeline-status ng-if="stage.status" status="stage.status"></pipeline-status>
+          <!-- <div class="pipeline-time" ng-if="stage.durationMillis">{{stage.durationMillis | humanizeDurationValue}}</div> -->
```
Comment (spadgett) [inline app/views/directives/build-pipeline.html]: This isn't new, but do you mind taking out the commented code while here?

**Comment by spadgett** [comment]
Just curious how it looks with an arrow instead of the right triangle.

This is definitely an improvement.

**Comment by rhamilto** [comment]
Arrived at this:

screen shot 2016-07-19 at 3 43 46 pm
screen shot 2016-07-19 at 3 43 25 pm
screen shot 2016-07-19 at 3 43 14 pm
screen shot 2016-07-19 at 3 43 03 pm

PR to follow.

**Comment by rhamilto** [comment]
@jwforres, please take a look.  This PR only addresses the display of stages as I'll need an assist on changing the display of pipelines on the overview page.

**Comment by rhamilto** [comment]
Fixes #113

**Comment by spadgett** [comment]
@rhamilto I have a feeling you won't like this comment... but the gap to the right of the last stage looks a little strange when none of my pipelines wrap :/

openshift_web_console

**Comment by rhamilto** [comment]
@spadgett, I wrestled with that, and I've not been able to come up with a solution that maintains stages that are equal widths without moving the position of the arrow like so:

screen shot 2016-07-21 at 9 40 28 am
screen shot 2016-07-21 at 9 40 43 am

I think the gap on the right is the lesser of two evils.  Thoughts?

**Comment by spadgett** [comment]
[merge]

**Pull request merged**